apps/web/examples/datepicker/datepicker.localization.tsx

```diff
@@ -6,21 +6,36 @@ const code = `
 
 import { Datepicker } from "flowbite-react";
 
+<<<<<<< HEAD
 export function Component() {
   return <Datepicker language="pt-BR" labelTodayButton="Hoje" labelClearButton="Limpar" />;
+=======
+function Component() {
+  return <Datepicker language="ptBR" labelTodayButton="Hoje" labelClearButton="Limpar" />;
+>>>>>>> e9b5fa9 (now supporting localization with supported locaziation from date-fns)
 }
 `;
 
 const codeRSC = `
 import { Datepicker } from "flowbite-react";
 
+<<<<<<< HEAD
```

Comment:
You forgot to resolve the conflicts

Reply:
oh no, sorry... let me go thorugh again, and try to fix all